.env.example

```diff
@@ -61,7 +61,6 @@ SPRINT_SIZE_BLOCK_NUMBER=0 # comma separated values defining the block heights o
 BLOCK_NUMBER=0 # comma separated values defining the block heights of bor where block time must change
 BLOCK_TIME=2 # comma separated values defining the block times for the relative BLOCK_NUMBERs
 BOR_REPO="https://github.com/maticnetwork/bor.git" # repo of bor project
-# BOR_REPO="https://<username>:<token>@github.com/<username>/<repo>.git" # example of private repo URL
```

Comment:
Please keep this into the example, as it is useful when working with private repos (e.g. security fixes)